docusaurus/docs/iOS/uikit/navigation.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Navigation
+---
+
+The StreamChatUI SDK not only provides reusable UI Components, but also it comes with navigation flows out of the box, that you can customize.
+
+## Routers
+
+The navigation of StreamChatUI SDK is handled by `NavigationRouter`'s. Currently, the available routers in the SDK are the following:
+
+- `ChatChannelListRouter`: The Channel List Router is responsible to handle the navigation of the Channel List, like showing the Channel View.
+- `ChatMessageListRouter`: The Message List Router is responsible to handle the navigation of the Message List, like showing the User Profile View, the Gallery View, the Message Actions Popup, etc.
+- `AlertsRouter`: The Alerts Router is responsible to show alerts in the Chat SDK, like confirmation dialogs.
+
+### Presenting a custom user profile view
+
+Currently, the StreamChatUI SDK does not come with a user profile view out of the box, since most apps already have their own user profile view. If you want to show your user profile view whenever the the user clicks on an avatar or clicks on a user mention, you can do so my customizing the `ChatMessageListRouter`.
```

Comment:
```suggestion
Currently, the StreamChatUI SDK does not come with a user profile view out of the box, since most apps already have their own user profile view. If you want to show your user profile view whenever the user clicks on an avatar or clicks on a user mention, you can do so my customizing the `ChatMessageListRouter`.
```